test/parallel/test-spawn-cmd-named-pipe.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+
+// This test is intended for Windows only
+if (!common.isWindows) {
+  console.log('1..0 # Skipped: this test is Windows-specific.');
+  return;
+}
+
+if (!process.argv[2]) {
+  // parent
+  const net = require('net');
+  const spawn = require('child_process').spawn;
+  const path = require('path');
+
+  const pipeNamePrefix = path.basename(__filename) + '.' + process.pid;
+  const stdinPipeName = '\\\\.\\pipe\\' + pipeNamePrefix + '.stdin';
```

Comment:
Thanks for the suggestion. I assume that the `\\\\?\\pipe` syntax is to support extended-length names. I don't think it should matter much here as the pipe name is hardcoded.